### PR TITLE
Log warning when regression model has predict_proba()

### PR DESCRIPTION
**Pull request by gaugup**

### CODE REVIEW

**Review by gregorybchris — APPROVED** [review]

File: responsibleai/responsibleai/modelanalysis/model_analysis.py [hunk 1]
```diff
@@ -309,6 +309,13 @@ def _validate_model_analysis_input_parameters(
                             ' getting predictions via predict_proba()'
                         )
 
+                if task_type == ModelTask.REGRESSION:
```
Comment (gregorybchris) [inline responsibleai/responsibleai/modelanalysis/model_analysis.py:312]: Might consider making this an `elif` of the condition above. Otherwise looks good

Comment (gaugup) [reply responsibleai/responsibleai/modelanalysis/model_analysis.py:312]: Keeping this as a spearate if.

File: responsibleai/responsibleai/modelanalysis/model_analysis.py [hunk 2]
```diff
@@ -309,6 +309,13 @@ def _validate_model_analysis_input_parameters(
...
+                    if hasattr(model, 'predict_proba'):
```
Comment (imatiach-msft) [inline responsibleai/responsibleai/modelanalysis/model_analysis.py:313]: can you use the constant from here:

https://github.com/microsoft/responsible-ai-widgets/blob/850bf429f20b5401b7907e249d219dbd5d97f6c9/responsibleai/responsibleai/_internal/constants.py#L56

File: responsibleai/responsibleai/modelanalysis/model_analysis.py (hunk 1 above)
Comment (imatiach-msft) [reply responsibleai/responsibleai/modelanalysis/model_analysis.py:312]: nit: I think using and is better than two ifs, but please ignore if you prefer this way

```if task_type == ModelTask.REGRESSION and hasattr(model, PREDICT_PROBA):```

**Review by imatiach-msft — APPROVED** [review]

**Pull request merged**